.github/workflows/ci.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         exclude:
           - os: ubuntu-latest 
             msvc: false
+          - os: windows-latest 
```

Comment:
@Tectu This is what disables mingw. Removing these two lines will restore it building both mingw and msvc